CMakeLists.txt

```diff
@@ -1099,11 +1115,17 @@ ENDIF(PCRE2_BUILD_TESTS)
 # Installation
 
 SET(CMAKE_INSTALL_ALWAYS 1)
+set(TARGETS_EXPORT_NAME pcre2-targets)
```

Comment:
I don't see a benefit from this variable/constant. It is just a redirection.

Reply:
Inlined.